app/routes/chatbot.js

```diff
@@ -13,13 +12,24 @@ const phoenix = require('../../lib/phoenix');
 const stathat = require('../../lib/stathat');
 const ClosedCampaignError = require('../exceptions/ClosedCampaignError');
 
+// config modules
+const requiredParamsConf = require('../../config/middleware/chatbot/required-params');
+const requestContextConf = require('../../config/middleware/chatbot/request-context');
```

Comment:
This is exactly the place where I did some soul searching about the name. I agree the naming is generic. I was thinking something like `request-conversation-context`? The `request-input-params` is better than what is there now, but still generic. The `requiredParamsConf ` middleware takes care of making sure required params are there, for now `phone` is the only non-negotiable param. We then look for one of 3: `args`, `keyword`, or `mms_image_url`. It seems to me that, the "context" or this request/conversation is defined by which of these params is included in the request, thats why i named it that way.